.test-infra/jenkins/CommonLoadTestConfiguration.groovy

```diff
@@ -0,0 +1,216 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import CommonTestProperties
+
+class CommonLoadTestConfiguration {
```

Comment:
I wonder if it would be better if we have the `CommonLoadTestConfiguration`, streaming testing job and batch testing job in one file? We can create multiple jobs in one file. They could reuse `CommonLoadTestConfiguration` the same way but without the need to import it. The config would also remain hidden for the load tests only - after all we wouldn't want other (non load-test) jobs to be able to import this. WDYT?